orca-front50/src/main/groovy/com/netflix/spinnaker/orca/front50/model/PluginArtifact.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2020 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.orca.front50.model;
+
+import java.util.List;
+import java.util.regex.Pattern;
+import javax.annotation.Nonnull;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class PluginArtifact {
+  @Nonnull private String id;
+  private String description;
+  private String provider;
+  @Nonnull private List<Release> releases;
+  private Long createTs;
+  private Long lastModified;
+  private String lastModifiedBy;
```

Comment:
1. `createTs` - Is this intended to be when the artifact was published? If that's the case, then it should be something like `publishedAt`.
1. All 3 of these fields are private as far as front50 is concerned and the values cannot be affected by outside data - I don't think we need them.
1. The codebase is a huge mixed bag on this because legacy, but our modern preference is to represent all timestamps as `Instant`.

Reply:
I wanted these fields so that when we get the PluginArtifact back from front50 after the upsert we can add the artifact to the task outputs and see the exact PluginArtifact that was upserted.  Of course, I'm not expecting us to pass these fields to front50 when we do the upsert.  Maybe not necessary?

Reply:
Not sure if all the audit type fields that are private to Front50 are required to be in Orca. Wouldn't hurt but may be not necessary.

Reply:
Alrighty, I'll remove them - we can always add them back if we want to see this data in the task outputs.